docs/content/rn-known-issues-ref.adoc

```diff
@@ -3,4 +3,5 @@
 
 This section lists known issues with {PRODUCT_SHORT} {PRODUCT_VERSION}.
 
-* Developer users and application users are currently defined in the same Red Hat Single Sign-On instance. This will be resolved in a future release.
+* You cannot create persistent queues using AMQ Online. This issue will be addressed in a future release.
+* You do not have the required privileges to add walkthroughs to a cluster. If you require this feature, contact Red Hat Support for a workaround.
```

Comment:
Do Red Hat Support have a workaround?

Reply:
options include giving permission to user